.remarkrc.js

```diff
@@ -0,0 +1,6 @@
+// Configuration file used to set up a tool to find mistakes in .md files.
```

Comment:
```suggestion
// Configuration file used to find popular mistakes in .md files.
```

Let’s make it shorter.